contributors/devel/conformance-tests.md

```diff
@@ -1,110 +1,150 @@
-- cannot rely on any binaries that are not required for the
-linux kernel or for a kubelet to run (i.e. git)
-- any container images used in the test must support all architectures for which kubernetes releases are built
-
-### Conformance Test Version Skew Policy
+The Kubernetes Conformance test suite is a subset of e2e tests that SIG
+Architecture has approved to define the core set of interoperable features that
+all conformant Kubernetes clusters must support. The tests verify that the
+expected behavior works as a user might encounter it in the wild.
+
+The process to add new conformance tests is intended to decouple the development
+of useful tests from their promotion to conformance:
+- Contributors write and submit e2e tests, to be approved by owning SIGs
+  - Tests should begin by covering the most important and visible aspects of the
+    function
+- Tests are proven to meet the [conformance test requirements] by review
+  and by accumulation of data on flakiness and reliability
+- A follow up PR is submitted to [promote the test to conformance](#promoting-tests-to-conformance)
+
+NB: This should be viewed as a living document in a few key areas:
+- The desired set of conformant behaviors is not adequately expressed by the
+  current set of e2e tests, as such this document is currently intended to
+  guide us in the addition of new e2e tests than can fill this gap
+- This document currently focuses solely on the requirements for GA,
+  non-optional features or APIs. The list of requirements will be refined over
+  time to the point where it as concrete and complete as possible.
+- There are currently conformance tests that violate some of the requirements
+  (ie: require privileged access), we will be categorizing these tests and
+  deciding what to do once we have a better understanding of the situation
+- Once we resolve the above issues, we plan on identifying the appropriate areas
+  to relax requirements to allow for the concept of conformance Profiles that
+  cover optional or additional behaviors
+
+## Conformance Test Requirements
+
+Conformance tests currently test only GA, non-optional features or APIs. More
+specifically, a test is eligible for promotion to conformance if:
+
+- it tests only GA, non-optional features or APIs (ie: no alpha or beta
+  endpoints, no feature flags required, no deprecated features)
+- it works for all providers (ie: no `SkipIfProviderIs`/`SkipUnlessProviderIs`
+  calls)
+- it is non-privileged (ie: no root on nodes, network, nor cluster)
+- it works without access to the public internet
+- it works without non-standard filesystem permissions granted to pods
+- it does not rely on any binaries that would not be required for the linux
+  kernel or kubelet to run (ie: can't rely on git)
+- any container images used within the test support all architectures for which
+  kubernetes releases are built
+- it passes against the previous release of kubernetes as well as the current
```

Comment:
Is this consistent with the table below where it talks about the previous 2 releases, not just one? Or is there just missing an 's' on "release" in this sentence?

Reply:
Rewording to be more explicit, but this is consistent. v1.3.x is expected to pass against v1.3.x and v1.2.x, but not v1.1.x.